src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -487,7 +362,7 @@ struct
     let texpr1 = Texpr1.of_expr (A.env nd) (Var v') in
     A.substitute_texpr_with Man.mgr nd v texpr1 None
 
-  let meet_tcons d tcons1 =
+  let meet_with_tcons d tcons1 =
     let earray = Tcons1.array_make (A.env d) 1 in
     Tcons1.array_set earray 0 tcons1;
     A.meet_tcons_array Man.mgr d earray
```

Comment:
This isn't an in-place operation, so the `with` in the name is misleading.

Reply:
21a4e189ddb8aa968383badb06a5ae4b473a3dbb